lib/kube/kubeconfig/kubeconfig_test.go

```diff
@@ -253,7 +253,8 @@ func TestUpdateWithExec(t *testing.T) {
 				fmt.Sprintf("--kube-cluster=%s", kubeCluster),
 				fmt.Sprintf("--teleport-cluster=%s", clusterName),
 			},
-			Env: []clientcmdapi.ExecEnvVar{{Name: homeEnvVar, Value: home}},
+			Env:             []clientcmdapi.ExecEnvVar{{Name: homeEnvVar, Value: home}},
+			InteractiveMode: clientcmdapi.IfAvailableExecInteractiveMode,
```

Comment:
This change wasn't in the original PR, just highlighting it to double check that it's intentional.